transformers/algorithm.md

```diff
@@ -0,0 +1,40 @@
+# Algorithm
+This is the name given to the portion of a transformer that provides the analysis and/or transformation of data.
+
+Please read the [Transformers](https://github.com/AgPipeline/AgPipeline.github.io/blob/master/transformers/transformers.md) overview documentation for some additional context.
+
+In this document we will be providing an conceptual overview of the Algorithm code structure and its implementation framework for the UA makeflow solution.
+
+## Overview
+The use of the term Algorithm appears to be nebulous in that implementations can range from processing raw sensor data to performing time based analysis of derived data.
+The unifying principle for Algorithm implementation is the manipulation of data.
+For example, the Algorithm implementations for RGB processing in the AgPipeline includes converting the *.bin* files to a georeferenced *.tif* file, masking soil from the TIFFs, clipping images to plot boundaries, and calculating canopy cover from the plot-clipped TIFF data.
+
+By standardizing a minimal set of required file names and function signatures with return values, Algorithm implementations can more easily be used in different environments.
+Outside of satisfying these requirements, an Algorithm implementation is free to become as complicated as necessary to do its work.
```

Comment:
I think this is important but that we need to be more specific about what the requirements are (a common / standard set of inputs and outputs?)

Reply:
I'm not clear on what you're asking; isn't that covered in line 13? Or are you looking to have the specifics listed here?